### ISSUE TITLE
[EN][Hunter][Beast Mastery][Spirit Beast] - Autocast 

### ISSUE THREAD

**Issue by meowmewo** [Atlantiss/BugTracker#2728]
So i went BM a week ago, tamed a Spirit beast (arcturis) and i noticed something happening in raids as well as many times on Dummy. From what ive experienced it happens after i use Spirit Mend ability of my pet and after Bestial Wrath. What happens is my pet's special attacks that are on Autocast mode - (Claw etc)  pretty much just stop going off. So i have to re put autocast or call off and send in again my pet in order to fix that. Im sure this is happening and i ask ppl to tell me if they experience the same problem. Its not happening 100% of times but the more i play the more i noticed that and it really affects my gameplay in raid enviroment.

EDIT. In order to be sure of my findings upon making this post i logged and went to the heroic training dummy. I sent my pet, i casted Bestial and it happened. My pet stopped using Claw on autocast and continued to NOT USE it after BW ended.

**Comment by ShockRW** [comment]
This is strange, I've jest tested it and you are totally right. I used BW for the second time (because the first one seemed ok) and my pet stopped using Claw, pet's focus was full all the time. However, after BW expired my pet stil wasn't using it. It required to back and send pet again to the current target to make it work again. However I think this bug may be related to pets' focus regen, which is a lill bit bugged here. For example pet focus should regen "smoothly" like the hunter one, not periodicaly like it is at the moment, what's more, for some reason summoned pet does not start with 100% focus, but with zero. There is a report about it. 
https://github.com/Atlantiss/BugTracker/issues/436

**Comment by Wolffenstein** [comment]
Weird, I keep spamming Bestial Wrath and my pet is still casting Claw properly. Maybe it's a talent issue, can you check after resetting your talents?

**Comment by meowmewo** [comment]
All pet families have Wild Hunt in their talent trees. I just tested it with Cunning pet. Still bugging.

**Comment by meowmewo** [comment]
Yes i just tested after resseting. No change

**Comment by ShockRW** [comment]
Yeah all pet famillies have Wild Hunt Talent, my bad sorry. I meant that I didin't spec one of my current Cunning pet into Wild Hunt. I tested it like 5 times with cunning pet and I didin't notice a problem with claw during BW. However I did a lot with ferocity pet (Spirit Beast).

**Comment by meowmewo** [comment]
Speaking of Wild Hunt white testing i just noticed how its not working fully.

**Comment by smmuntz** [comment]
I know I have noticed this issue in the past, just never really thought anything of it at the time. I checked this out in game after reading this bug, and when testing against a Heroic Training Dummy I was able to get the bug to happen, but not reliably. I was switching between using Mend Pet, Bestial Wrath, and Call of the Wild and combinations of those but sometimes it would occur and other times it would work fine.

**Comment by meowmewo** [comment]
Player around and i found something interesting that triggers that basic attacks not going off bug like 70% of time..and guess what.. its Dash.
So when i send me pet with Dash, after it uses it the pet gets to the target and does not do any basic attacks. If i call back the pet and send it back again (It still has the Dash up) it starts using basic attacks. There was no Bestial Wrath use at any moment while i was testing. Just Dash and Claw

**Comment by meowmewo** [comment]
I mean it seems this bug is connected to many other abilities and its too deep. I rly cant filter what is the main problem..Guess we will have to macro a Hunter shot + Claw , Bite etc..so if the bug happens the autocast will be refreshed.

**Comment by lukipol** [comment]
complete joke bug my friends

**Comment by Tj20** [comment]
I added a "your pet isn't using special attacks" reminder to my [autoshot bug warning addon](http://atlantiss.eu/community/viewtopic.php?f=134&p=9710#p9710).
So far I haven't noticed anything unusual, but I play mainly SV atm, so no BM. Also I don't like Dash on auto cast, it seems to me like a waste of focus. The pet even casts it (=30 focus) when it can sit perfectly still on one spot.

**Comment by meowmewo** [comment]
Ye that was for the test.Personally i just use /cast Arcane Shot /cast Bite,Claw etc on a macro and since Arcane is part of any rotation if the bug ocurrs its instantly removed due to /cast Bite refreshing the autoattack. That autoshot bug tho..gotta be fixed asap.
